### ISSUE TITLE
Support for CentOS - a freely available clone of Red Hat Enterprise Linux

### ISSUE THREAD

**Issue by jfautley** [Arachnid/netboot.me#32]
Hi,

I've created configs for CentOS 5, 32 and 64bit, plus rescue mode. These are:

http://www.netboot.me/229002 - 5/i386/installer
http://www.netboot.me/242001 - 5/x86_64/installer

http://www.netboot.me/243001 - 5/i386/rescue
http://www.netboot.me/229003 - 5/x86_64/rescue

Adding CentOS 4 support is as simple as replacing the references to '5' with '4'. You could also use specific update versions, i.e. for 5.5 replace '5' with '5.5' in the URLs, but I see no real reason to do this.

Cheers,

Jon

**Comment by dotwaffle** [comment]
Thanks Jon, I'll get testing these right away. Can you confirm whether CentOS 4 is "current" - ie, is it a deprecated distro, or likely to be supported for some time?

**Comment by jfautley** [comment]
Yeah, supported until 29th Feb 2012 - http://www.redhat.com/security/updates/errata/
(that's obviously the RHEL support policy, but CentOS track that)

Probably worth including it for completeness. No point with 3 though - that dies on 31st Oct. 2010 :)
